src/transformers/configuration_utils.py

```diff
@@ -578,30 +581,50 @@ def get_config_dict(
                 use_auth_token=use_auth_token,
                 user_agent=user_agent,
             )
-            # Load config dict
-            config_dict = cls._dict_from_json_file(resolved_config_file)
 
+        except RepositoryNotFoundError as err:
+            logger.error(err)
+            raise EnvironmentError(
+                f"{pretrained_model_name_or_path} is not a local folder and is not a valid model identifier listed on "
+                "'https://huggingface.co/models'\nIf this is a private repository, make sure you are properly logged "
+                "in with a token having permission to this repo."
```

Comment:
```suggestion
                "in with a token having permission to this repo (`huggingface-cli login`).
```
(nit) maybe nice to have the login command right there

Reply:
Actually the token needs to be passed explicitly or with `use_auth_token=True`, will amend.